sdk/storage/azure-storage-file-datalake/tests/test_file.py

```diff
@@ -859,10 +859,10 @@ def test_rename_file_with_file_sas(self, datalake_storage_account_name, datalake
         self.assertEqual(data, data_bytes)
         self.assertEqual(new_client.path_name, "newname")
 
+    @pytest.mark.skip(reason="Service bug")
```

Comment:
Do we know what the bug is? Or does this still just need more investigation?

Reply:
Added more descriptive skip; was originally a `pytest.skip` inside so I just moved it outside to use the decorator `pytest.mark.skip`. No idea on what the bug is unfortunately 😞